src/Message/CaptureResponse.php

```diff
@@ -45,6 +45,7 @@ public function getRedirectUrl()
 
 	public function getRedirectMethod()
 	{
+		// FIXME: A redirect should nearly always be a GET. Is this really correct?
 		return 'POST';
```

Comment:
This must actually be POST with this gateway, when redirecting to the off-site payment form